index.html

```diff
@@ -718,24 +736,26 @@ <h3>Resource Timing Attributes</h3>
 <h2>Creating a resource timing entry</h2>
 <p>To <dfn export="">mark resource timing</dfn> given a
 [=/fetch timing info=] |timingInfo|, a DOMString
-|requestedURL|, a DOMString |initiatorType| and a <a>global object</a> |global|:
+|requestedURL|, a DOMString |initiatorType| a <a>global object</a> |global|, and a string |cacheMode|,
+perform the following steps:
 <ol>
 <li>Create a <a>PerformanceResourceTiming</a> object |entry| in |global|'s [=global object/realm=].
-<li><a>Setup the resource timing entry</a> for |entry|, given |initiatorType|, |requestedURL| and
-|timingInfo|.
+<li><a>Setup the resource timing entry</a> for |entry|, given |initiatorType|, |requestedURL|,
+|timingInfo|, and the empty string or "<code>local</code>" |cacheMode|.
 <li><dfn data-lt="step-final-queue"><a data-cite=
 "PERFORMANCE-TIMELINE-2#dfn-queue-a-performanceentry">Queue</a> |entry|.</li>
 <li><a data-lt='add a PerformanceResourceTiming entry'>Add</a> |entry| to |global|'s
 <a data-cite="PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance
 entry buffer</a>.</li>
 </ol>
 <p>To <dfn export="">setup the resource timing entry</dfn> for <a>PerformanceResourceTiming</a> |entry|
-given DOMStrring |initiatorType|, DOMString |requestedURL|, and [=/fetch timing info=] |timingInfo|,
-perform the following steps:</p>
+given DOMStrring |initiatorType|, DOMString |requestedURL|, [=/fetch timing info=] |timingInfo|, and
+the empty string or "<code>local</code>" |cacheMode|, perform the following steps:</p>
```

Comment:
I think you can replace the possible values with "DOMString", and maybe assert that it's one of the possible values below.